app/v2/config/AppConfig.scala

```diff
@@ -28,20 +30,34 @@ trait AppConfig {
   def desEnv: String
 
   def desToken: String
+
+  def confidenceLevelConfig: ConfidenceLevelConfig
 }
 
 @Singleton
-class AppConfigImpl @Inject()(config: ServicesConfig) extends AppConfig with FixedConfig{
+class AppConfigImpl @Inject()(config: ServicesConfig, configuration: Configuration) extends AppConfig with FixedConfig{
 
 
   val mtdIdBaseUrl: String = config.baseUrl("mtd-id-lookup")
   val desBaseUrl: String = config.baseUrl("des")
   val desEnv: String = config.getString("microservice.services.des.env")
   val desToken: String = config.getString("microservice.services.des.token")
+  val confidenceLevelConfig: ConfidenceLevelConfig = configuration.get[ConfidenceLevelConfig](s"api.confidence-level-check")
```

Comment:
This can just be a straight Boolean
```suggestion
  val authServiceCheckEnabled: Boolean = config.getBoolean(s"api.confidence-level-check. auth-validation.enabled")
```
or something